docs/General/Manuals/client-manual.rst

```diff
@@ -1480,6 +1480,171 @@ quick way to move around a large map in this manner.
+output segment and the city's full vision radius highlighted over the main map as highlighted in Figure 48.
+This city is size 3, so this means that 4 tiles can be worked by the citizens. The city center is worked for
+free and then 3 other tiles are being worked. In this example, there is a forest/river+silk tile,
+grassland+irrigation, and a grassland/river tile being worked. You can click on the city center and the client
+will automatically pick the best tiles for net food to aid city growth. You can also click on a tile to remove
+the citizen from the tile and then click another tile to have the citizen work another tile. This is commonly
+referred to a city micromanagement. If you remove a citizen from working a tile, take a look at the top center
+segment. The citizens icon bar will show one entertainer specialist. If you want to change the entertainer to
+a scientist or a taxman, you can click on it in the icon bar to change.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-center.png
+  :align: center
+  :alt: Freeciv21 City Dialog City Center
+  :figclass: align-center
+
+  Figure 48: City Dialog - City Center
+
+Moving right, we can see the :guilabel:`General` tab on the right most panel as highlighted in Figure 49. This
+tab shows information simiar to what is shown on the top center segment, along with units and city
+improvements that the city has produced and is supporting. Hovering your mouse over many of the items at the
+top of the :guilabel:`General` tab will show detailed calculation information on how the net value is
+produced.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-general.png
+  :align: center
+  :alt: Freeciv21 City Dialog General
+  :figclass: align-center
+
+  Figure 49: City Dialog - General
+
+
+We will discuss the :guilabel:`Citizens` tab in a bit. Moving to the bottom center, you will see a list of all
+the units that are present in the city as highlighted in Figure 50. There can be units present in a city that
+the city is not supporting, or from your allies. If you right-click on a unit, a pop up box shows some action
+commands that you can give to the unit. If you double-click on a unit, the :guilabel:`City Dialog` will close
+and the unit will be selected. See `Unit Controls`_ for more information on what to do with the unit.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-units.png
+  :align: center
+  :alt: Freeciv21 City Dialog Units
+  :figclass: align-center
+
+  Figure 50: City Dialog - Present Units
+
+
+Now let's come back and talk about the :guilabel:`Governor` tab on the left side segment and the
+:guilabel:`Citizens` tab on the right side segment.
+
+The :guilabel:`Governor` tab as highlighted in figure 51, gives information on the citizen management agent
+(CMA) for this city. For more information on how to use the CMA refer to :doc:`../../Playing/cma`.
```

Comment:
We should probably decide on a single name. Calling it both CMA and Governor seems redundant and confusing. I'd go with Governor.